### PR TITLE
EOS:15091 - Added System Tests related to source metadata load and validation.

### PR DESCRIPTION
**Pull request by amitwac2608**
Signed-off-by: Amit Kumar <amit.kumar@seagate.com>

### CODE REVIEW

**Comment by amitwac2608** [comment]
Jenkins Job: http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server-7.8.2003/183

**Review by bkirunge7 — COMMENTED** [review]
Add a test to cover below scenerio 
1. Initiate multipart, dont call complete multipart on destination bucket2 for object say sample.txt
2. Upload sample.txt to bucket1
3. Copyobject bucket1/sample.txt bucket2/

File: st/clitests/awss3api_spec.py [hunk 1]
```diff
@@ -369,24 +369,74 @@ def delete_object_list_file(file_name):
...
 
-# ************ CopyObject API supported *************************
+# ************ CopyObject API supported *****************************************************************
+
 AwsTest('Aws can put object').put_object("seagatebucket", "1kfile", 1024)\
     .execute_test().command_is_successful()
 
-AwsTest('Aws can copy object').copy_object("seagatebucket/1kfile", "seagatebucket", "1kfile-copy")\
+# Positive: copy-object from a bucket to the same bucket as destination.
+AwsTest('Aws can copy object to same bucket').copy_object("seagatebucket/1kfile", "seagatebucket", "1kfile-copy")\
```
Comment (knrajnambiar76) [inline st/clitests/awss3api_spec.py:378]: you can add test to copy file of 0 bytes

**Review by bkirunge7 — APPROVED** [review]

**Review by knrajnambiar76 — APPROVED** [review]

File: st/clitests/awss3api_spec.py (hunk 1 above)
Comment (amitwac2608) [reply st/clitests/awss3api_spec.py:378]: I have done the manual testing of this test. As discussed, not adding this test to system test file.

**Comment by amitwac2608** [comment]
> 
> 
> Add a test to cover below scenerio
> 
>     1. Initiate multipart, dont call complete multipart on destination bucket2 for object say sample.txt
> 
>     2. Upload sample.txt to bucket1
> 
>     3. Copyobject bucket1/sample.txt bucket2/

We will add this test case when the CopyObject API will do actual object oid creation and copy data.

**Pull request merged**